internal/runner/service_test.go

```diff
@@ -621,6 +621,96 @@ func Test_runnerService(t *testing.T) {
 		assert.Equal(t, "LICENSE: "+string(expected), string(result.Stdout))
 		assert.EqualValues(t, 0, result.ExitCode)
 	})
+
+	if _, ok := os.LookupEnv("GITHUB_ACTIONS"); !ok {
+		t.Run("ExecuteWinsizeDefault", func(t *testing.T) {
```

Comment:
This one might be moved to `command_test.go` as it does not test any gRPC service functionality unlike the two below.

Reply:
Yep that makes sense!